exec/csnex/Exec.cs

```diff
@@ -257,7 +259,8 @@ void PUSHI()
 
         void PUSHNIL()
         {
-            throw new NotImplementedException(string.Format("{0} not implemented.", MethodBase.GetCurrentMethod().Name));
+            ip++;
+            stack.Push(new Cell());
```

Comment:
The `NIL` value is actually a pointer value and you should probably use `Cell(null)` instead. There's a difference between pushing a cell that doesn't contain anything, versus pushing a cell that contains a pointer which happens to be `NIL`.

Reply:
Changed to `new Cell((object)null)` to avoid ambiguous call between `Cell.Cell(List<Cell>)`, and `Cell.Cell(byte[])`.  This will be cleaned up in the upcoming `csnex.Cell()` class clean up.